samples/drumthumper/src/main/java/com/google/oboe/sample/drumthumper/DrumThumperActivity.kt

```diff
@@ -125,46 +149,79 @@ class DrumThumperActivity : AppCompatActivity(), TriggerPad.DrumPadTriggerListen
         setContentView(R.layout.drumthumper_activity)
 
         // hookup the UI
-        run {
-            var pad: TriggerPad = findViewById(R.id.kickPad)
-            pad.addListener(this)
-        }
-
-        run {
-            var pad: TriggerPad = findViewById(R.id.snarePad)
-            pad.addListener(this)
-        }
-
-        run {
-            var pad: TriggerPad = findViewById(R.id.midTomPad)
-            pad.addListener(this)
-        }
-
-        run {
-            var pad: TriggerPad = findViewById(R.id.lowTomPad)
-            pad.addListener(this)
-        }
-
-        run {
-            var pad: TriggerPad = findViewById(R.id.hihatOpenPad)
-            pad.addListener(this)
-        }
-
-        run {
-            var pad: TriggerPad = findViewById(R.id.hihatClosedPad)
-            pad.addListener(this)
-        }
-
-        run {
-            var pad: TriggerPad = findViewById(R.id.ridePad)
-            pad.addListener(this)
-        }
-
-        run {
-            var pad: TriggerPad = findViewById(R.id.crashPad)
-            pad.addListener(this)
-        }
-
+        var sb : SeekBar;
+
+        // "Kick" drum
+        (findViewById(R.id.kickPad) as TriggerPad).addListener(this)
```

Comment:
If you use view bindings you can avoid using `findViewById`: https://developer.android.com/topic/libraries/view-binding

Reply:
I will investigate.

Reply:
I'm handling it with findViewById<TriggerPad>(....). I'd rather not tackle bindings right now.